src/test/java/org/innowise/OrderAnalyticsTest.java

```diff
@@ -0,0 +1,66 @@
+package org.innowise;
+
+import org.innowise.model.*;
+import org.innowise.service.OrderAnalytics;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class OrderAnalyticsTest {
+    private Customer customer1;
+    private Customer customer2;
+    private Customer customer3;
+    private List<Order> orders;
+
+    @BeforeEach
+    void setUp(){
```

Comment:
No tests for empty order lists
No tests for multiple orders with the same status
No test with a customer having more than 5 orders